modules/mobile/spec/request/immunizations_request_spec.rb

```diff
@@ -42,292 +42,309 @@
         # expect().to match_json_schema('immunizations')
         expect(response.parsed_body).to eq(
           { 'data' =>
-            [{ 'id' => 'I2-A7XD2XUPAZQ5H4Y5D6HJ352GEQ000000',
```

Comment:
Made some changes to the data for get_immunizations.yml. Including: 

- added manufacturer and group name fields (not all immunizations have manufacturer, but it's not only confined to covid vaccines)
- added the duplicate vaccine that will get filtered out from Jaysons duplicate vaccine issue (the second entry is now a duplicate of the first with just a different id) 
- removed the manufacturer for one of the covid vaccines to test it would be processed properly.

Other vcr changes are just added the group name and manufacturer when applicable but just for schema consistency. Those changes didn't effect the logic of the tests using the immunization blank note, cache token, etc..